docs/user/tutorial/30-structure-content-modes.rst

```diff
@@ -43,19 +44,20 @@ Add a second plugin
 
 Let's add another plugin, containing a map.
 
-.. todo:: add the "Add plugin" icon as add-plugin-icon.png
+.. todo:: Daniele please check add the "Add plugin" icon as add-plugin-icon.png
 
-.. |add-plugin-icon| image:: /user/tutorial/images/add-plugin-icon.png
+.. image:: /user/tutorial/images/add-plugin-icon.png
    :alt: 'add plugin'
+   :width: 300
 
 #.  Select the **Add plugin** icon |add-plugin-icon|.
```

Comment:
This isn't rendering properly.

The end result should be:

>  1. Select the **Add plugin** icon **+**

(i.e. just the + icon, inline in the text).

And I think that there should *also* be the existing image that provides more context, below it.